### PR TITLE
[TR] Allow to select the Behat feature dirs or files to run on CI

### PR DESCRIPTION
**Pull request by BitOne**
This PR allows to select which Behat features dir or feature file to run on the PR. Multiple values can be provided.
Example:
locale:product/download_product.feature:channel
=> will run all features from features/locale and features/channel, as well as the features/product/download_product.feature file

| Q                  | A
| ------------------ | ---
| Migration script?  | N.A.
| Behats             | OK
| Specs & Static     | N.A.
| Changelog updated  | N.A. (internal tooling)

### CODE REVIEW

**Comment by sumbobyboys** [comment]
GTMMMMMMMMMMMM

File: build.xml [hunk 1]
```diff
@@ -317,13 +317,19 @@
...
+            <arg value="-c" />
+            <arg value="${behat.concurrency}" />
+            <arg value="-x" />
+            <arg value="${behat.xdebug}" />
+            <arg value="-d" />
+            <arg value="${symfony.database.name}_" />
+            <arg value="-p" />
+            <arg value="${behat.profile.prefix}-" />
+            <arg value="-f"/>
+            <arg value="${behat.features} "/>
```
Comment (jmleroux) [inline build.xml]: why a trailing whitespace ?

File: app/Resources/jenkins/behat/run_behat.sh [hunk 2]
```diff
@@ -11,47 +11,83 @@ CHECK_WAIT="0.5"
...
+    case $1 in
+        -x)     shift
+                eval $IS_OPTION || XDEBUG=$1
+                [ -n "$XDEBUG" ] && shift
+                ;;
+
+
+        -c)     shift
+                eval $IS_OPTION || CONCURRENCY=$1
+                [ -n "$CONCURRENCY" ] && shift
```
Comment (jmleroux) [inline app/Resources/jenkins/behat/run_behat.sh]: maybe check for max value ?

**Comment by jmleroux** [comment]
:+1: 
GTM after fix https://github.com/akeneo/pim-community-dev/pull/3201#discussion_r38002187

File: build.xml (hunk 1 above)
Comment (BitOne) [inline build.xml]: Needed for when features are empty to avoid mis-handling by ant argument management.

**Pull request merged**